packages/ListBox/src/helpers/DOMAttributes.js

```diff
@@ -0,0 +1,41 @@
+import uuidv4 from "uuid/v4";
+
+const listBoxId = `listBoxId_${uuidv4()}`;
+
+export function getDOMAttributesForListBoxContainer() {
+  return { tabIndex: "-1" };
+}
+
+export function getDOMAttributesForListBox(state) {
+  // NOTE: replace this with L10n component
```

Comment:
Why do you need the L10n here?

Reply:
I don't think we need it might be a comment when I started building the component